### ISSUE TITLE
The height is always 200px

### ISSUE THREAD

**Issue by vinhtq** [FormidableLabs/nuka-carousel#117]
I do not know why but the height seem always be 200px.

Even the picture is taller than 200px

```
<Carousel initialSlideHeight={300} width="250px" slidesToShow={1} wrapAround={true}>
     <img src="https://unsplash.it/300/400"/>
     <img src="https://unsplash.it/300/400"/>
     <img src="https://unsplash.it/300/400"/>
</Carousel>
```

The slider I put it in Modal. Does that make the height cannot be determined?

**Comment by lauterry** [comment]
Hello

Can you try with a single img in the Carousel to see if the issue still occured ?

**Comment by kenwheeler** [comment]
If you resize the window, does the height correct?

**Comment by vinhtq** [comment]
@lauterry If I put only one image, the carousel will adapt to the height of the image.

```initialSlideHeight``` property will be ignored

**Comment by tvarner** [comment]
Does anyone know of/have a way to implement adaptive height?

**Comment by kellyrmilligan** [comment]
I have the same issue as well. I have slides of varying height.

**Comment by kenwheeler** [comment]
This is getting fixed soon. I have to figure out a good way to give the container height without explicitly setting it.

**Comment by tvarner** [comment]
Hi Ken, thanks again for the Carousel. Really useful. I'd really like to use it for one of my projects, but the slide height issues are preventing me from doing so. Any progress on adaptive height? Thanks :)

**Comment by aronlmin** [comment]
Has a height setting been implemented yet? I am experiencing issues where my image sizes are 800px in height but they are being blown up to 1800

**Comment by MitchEnfield** [comment]
Any update? Or is the vertical height always 200px regardless of img size?

**Comment by ryan-roemer** [comment]
Is this still an issue on `nuka-carousel@3.1.0+` now that we've landed https://github.com/FormidableLabs/nuka-carousel/pull/291?

**Comment by ashika01** [comment]
fixed in #408  - release v4.3.9
